Maquetado/scss/titles.scss

```diff
@@ -0,0 +1,23 @@
+$gray-dark: #333;
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+$font-roboto: 'Roboto', sans-serif;
+
+
+.title {
+  align-items: center;
+  background: url(assets/triangle.svg) no-repeat;
+  display: flex;
+  height: 50px;
+  padding-left: 30px;
+  
+}
+h2 {
```

Comment:
leave an empty line between selectors and use classes for styles. move this to the `.title` class